src/main/java/com/woowacamp/soolsool/core/member/service/MemberService.java

```diff
@@ -21,26 +30,49 @@ public class MemberService {
 
     private final MemberRepository memberRepository;
     private final MemberRoleRepository memberRoleRepository;
+    private final MemberMileageChargeRepository memberMileageChargeRepository;
 
     @Transactional
     public void addMember(final MemberAddRequest memberAddRequest) {
-        final MemberRole memberRole = memberRoleRepository.findById(1L)
-            .orElseThrow(() -> new SoolSoolException(MemberErrorCode.MEMBER_NO_ROLE_TYPE));
+        checkDuplicatedEmail(memberAddRequest.getEmail());
+
+        final MemberRole memberRole = getMemberRole(memberAddRequest);
```

Comment:
```suggestion
        final MemberRole memberRole = getMemberRole(memberAddRequest.getMemberRoleType());
```

private method에는 DTO가 아닌 원시값을 넘기는게 활용도가 높을 것 같아요. 다른 DTO를 사용하는 서비스 메서드가 사용할 수 있으니...?

Reply:
굳굳~~